lib/thumbnail_utils.rb

```diff
@@ -96,7 +95,7 @@ def define_thumbnail_common(obj, provider)
     	else
     		abort "Invalid common repository type - #{provider.common_repository_type}"
   	end
-    asset_url
+    asset_url.downcase
```

Comment:
This would break Villanova thumbnails.
[Example working thumbnail](https://digital.library.villanova.edu/files/vudl:258991/THUMBNAIL)
[Example broken thumbnail with downcased path](https://digital.library.villanova.edu/files/vudl:258991/THUMBNAIL)